sdk/python/kfp_tekton/compiler/_op_to_template.py

```diff
@@ -133,20 +133,44 @@ def _op_to_template(op: BaseOp):
     outputs_dict = _outputs_to_json(op, processed_op.outputs, param_outputs, output_artifacts)
     if outputs_dict:
         template['spec']['results'] = []
+        outfile_step = {'command': ['/bin/bash'], 'image': 'ubuntu', 'name': 'outfile-step', 'args': ['-c', 'echo Copying files to Tekton dir...;']}
```

Comment:
can you break this into separate line for readability and rename `'outfile-step'` the step `'copy-results'` or `'copy-output-file'` (also slightly updated `echo`):

```Python
        outfile_step = {
            'command': ['/bin/bash'],
            'image': 'ubuntu',
            'name': 'copy-results',
            'args': ['-c', 'echo "Copying output files to Tekton results folder";']
        }
```